food-truck/src/app/pages/create-event/create-event.component.ts

```diff
@@ -1,5 +1,8 @@
 import { Component, OnInit } from '@angular/core';
 import { FormBuilder, FormGroup, FormArray } from '@angular/forms';
+import { MapsAPILoader } from '@agm/core';
+import {} from '@types/googlemaps';
```

Comment:
I think the right syntax here is `import` * from  '@types/googlemaps`

Reply:
this was recommended for agm and stackoverflow for google maps api.